src/descend.ml

```diff
@@ -1470,15 +1470,16 @@ value print conf base p =
   let templ =
     match p_getenv conf.env "t" with
     [ Some ("F" | "L" | "M") -> "deslist"
+    | Some ("H" | "I" | "A") -> "destable"
     | Some _ -> ""
     | _ -> "desmenu" ]
   in
   if templ <> "" then Perso.interp_templ templ conf base p
   else
     match (p_getenv conf.env "t", p_getint conf.env "v") with
-    [ (Some "A", Some v) -> print_aboville conf base v p
```

Comment:
I don't understand why you had to change the name ? It breaks the backward compatibility ?